make/manifest.py

```
@@ -110,38 +119,50 @@ def command(args: Namespace) -> None:
         b62 = hex_to_base62(digest)
         if b62 not in digest_info:
             digest_info[b62] = ([], digest)
 
         digest_info[b62] = (digest_info[b62][0] + [tag], digest)
 
     labels: dict[str, str] = {}
-    for b62, (tags, digest) in progress_bar(
-        digest_info.items(), prefix="Generating tag labels:" + " " * 4
-    ):
+    print("Generating tag labels...")
+    for b62, (tags, digest) in digest_info.items():
```

Ruff (0.15.9)

[warning] Loop control variable `b62` not used within loop body

(B007)